### PR TITLE
ALBERT Tokenizer integration test

### PR DESCRIPTION
**Pull request by LysandreJik**
Implements an integration test for the ALBERT tokenizer.

### CODE REVIEW

File: tests/test_tokenization_albert.py [hunk 1]
```diff
@@ -102,3 +102,29 @@ def test_sequence_builders(self):
...
+            tokenizer = tokenizer_class.from_pretrained("albert-base-v2")
+
+            sequence = "ALBERT: A Lite BERT for Self-supervised Learning of Language Representations"
+
+            encoding = tokenizer(sequence)
+            decoded_sequence = tokenizer.decode(encoding["input_ids"])
+
+            expected_encoding = {
+                "input_ids": [2, 2953, 45, 21, 13, 10601, 11502, 26, 1119, 8, 8542, 3762, 69, 2477, 16, 816, 18667, 3],
+                "token_type_ids": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
```
Comment (patrickvonplaten) [inline tests/test_tokenization_albert.py:122]: Maybe make a batched integration test to make it a bit more aggressive and make sure `atteniton_mask ` is correctly created?

**Comment by LysandreJik** [comment]
Good point!

File: tests/test_tokenization_albert.py [hunk 2]
```diff
@@ -102,3 +102,48 @@ def test_sequence_builders(self):
...
+        for tokenizer_class in tokenizer_classes:
+            tokenizer = tokenizer_class.from_pretrained("albert-base-v2")
+
+            sequences = [
+                "ALBERT: A Lite BERT for Self-supervised Learning of Language Representations",
+                "ALBERT incorporates two parameter reduction techniques",
+                "The first one is a factorized embedding parameterization. By decomposing the large vocabulary embedding matrix into two small matrices, we separate the size of the hidden layers from the size of vocabulary embedding."
+            ]
+
+            encoding = tokenizer(sequences, padding=True)
```
Comment (patrickvonplaten) [inline tests/test_tokenization_albert.py:121]: (nit & question) what is the recommend way of handling `padding`? Isn't it rather `padding="longest"`?

**Review by patrickvonplaten — APPROVED** [review]
Great, thanks for adding the test!

File: tests/test_tokenization_albert.py (hunk 2 above)
Comment (LysandreJik) [reply tests/test_tokenization_albert.py:121]: `True` is `longest`, `False` is `do_not_pad`, and then there's `max_length`. Either of those 5 work, and they're all mentioned in the docs so I would argue they're all recommended.

**Pull request merged**